components/plan-build/bin/hab-plan-build.sh

```diff
@@ -1997,7 +1997,24 @@ do_build_config() {
 do_default_build_config() {
   build_line "Writing configuration"
   if [[ -d "$PLAN_CONTEXT/config" ]]; then
-    cp -r "$PLAN_CONTEXT/config" $pkg_prefix
+    if [ -z "${HAB_CONFIG_EXCLUDE:-}" ]; then
+      # HAB_CONFIG_EXCLUDE not set, use defaults
+      config_exclude_exts=("*.sw*" "*~" "*.bak")
```

Comment:
Would an alternative to a set default be no set default? This would allow users of their editor to tweak for themselves. I'd tend to agree with @adamhjk here (it feels too broad), however in this specific case (filtering entries under the `config/` directory) I'm more okay with the chances of false positives.

It might make more sense to take @smith's suggestion and change `*.sw*` to `.sw?`.